compiler/extccomp.nim

```diff
@@ -667,7 +667,6 @@ proc addExternalFileToCompile*(conf: ConfigRef; filename: AbsoluteFile) =
 proc getLinkCmd(conf: ConfigRef; output: AbsoluteFile,
```

Comment:
how about insead add an optional param `removeStaticFile = true` to `getLinkCmd` and in `writeJsonBuildInstructions` call `getLinkCmd` with `removeStaticFile = false`